ambry-store/src/main/java/com.github.ambry.store/HardDeleter.java

```diff
@@ -464,8 +464,6 @@ void shutdown() throws InterruptedException, StoreException, IOException {
         }
         shutdownLatch.await();
         logger.info("Hard delete shutdown complete for store {} ", dataDir);
-        pruneHardDeleteRecoveryRange();
-        persistCleanupToken();
```

Comment:
same above

Reply:
same here, you can remove `StoreException, IOException` in method signature

Reply:
explained.